README.md

```diff
@@ -70,3 +70,62 @@ The two above might be also used together:
 - name: Publish my-crate to crates.io
   run: # your `cargo publish` code here
 ```
+
+# Customizing baseline rustdoc caching strategy
+
+The action caches the baseline rustdoc for each package in the workspace. The keys used to distinguish the caches consist of four components:
+
+ - `prefix-key` input, which defaults to an empty string,
+ - `shared-key` input if provided, otherwise a concatenation of the value of environmental variable `GITHUB_JOB`, the value of `package` and the hash of `manifest-path`,
```

Comment:
Just a suggested clarification, to avoid confusion and ambiguity.
```suggestion
 - `shared-key` input if provided, otherwise a concatenation of the value of environmental variable `GITHUB_JOB`, the value of `package` and the hashed value of the `manifest-path` variable (not the file it points to),
```

Reply:
I like it, it's definitely more clear now ;)